Beginners_guide_to_farm.md

```diff
@@ -152,11 +154,11 @@ c. Now lets make 2 folders we referenced earlier in the .sh and .R scripts. You
 d. Enter the command `ls`. You should still be in your home directory, and this will show all the files/folders in there. You should see two items: `myproject` and `MyPackages`.
 
 #####Moving files between farm and your computer
-a. To do this, you'll need to log out of farm - use the `exit` command. If you were in the home directory on your PC before logging on (/c/Users/Username/ in Windows), you should still be there - use `pwd` to check.
+a. To do this, you'll need to log out of farm - use the `exit` command. If you were in the home directory on your PC before logging on (`/c/Users/Username/` in Windows, `~` on Mac or unix-like), you should still be there - use `pwd` to check.
```

Comment:
@VadimKar so windows changed the way they specify directories? (last I used they started with `C:\`